### PR TITLE
Feature/daylight saving page

### PR DESCRIPTION
**Pull request by simonschwartz**
The PR adds custom templates for Qld and Tas daylight saving pages. It also includes the custom css for these pages.

This closes JIRA-406

daylight-saving

### CODE REVIEW

**Comment by ltankey** [comment]
Looks good! Just need to update CHANGELOG.md

**Comment by tessereth** [comment]
LGTM

**Comment by ltankey** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -18,6 +18,8 @@ This file is influenced by http://keepachangelog.com/.
 - Added published_at field to all pages
 - All pages now have short_summary fields
 - News article metadata can be edited
+- Added public holiday and daylight saving pages (static)
+- Added public holiday and daylight saving pages (static)
```
Comment (tessereth) [inline CHANGELOG.md]: Why is this in here twice?

**Pull request merged**